src/reducers/entities/plugins.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {combineReducers} from 'redux';
+import {PluginTypes} from 'action_types';
+
+function getInitialState() {
+    return {
+        data: [],
+        serverError: null,
+    };
+}
+
+function marketplacePlugins(state = getInitialState(), action) {
+    switch (action.type) {
+    case PluginTypes.RECEIVED_MARKETPLACE_PLUGINS: {
+        return {
+            ...state,
```

Comment:
nit: Seems unnecessary as all the state's fields are overridden anyways

Reply:
Fixed